conference/react-js/src/components/Tabs/Tabs.jsx

```diff
@@ -1,53 +1,125 @@
 import React from 'react';
-import { Route } from 'react-router-dom';
+import { Route, Switch, NavLink } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+
 import Upcoming from '../Upcoming';
-import AutorizedUserMenu from '../Autorized-user-menu';
+import NavigationBar from '../NavigationBar/NavigationBar';
+import Past from '../Past/Past';
+import ManageUsers from '../../containers/ManageUsers/ManageUsers';
+import Talks from '../../containers/Talks';
 import TalksWrapper from '../Talks/TalksWrapper/TalksWrapper';
 import {
   upcoming,
   myTalks,
   myEvents,
+  manageUsers,
+  talks,
+  past,
   conference } from '../../constants/route-url';
+import { admin } from '../../constants/roles';
 import MyEvents from '../MyEvents/MyEvents';
 import Conference from '../../containers/Conference';
 
-const Tabs = ({ user: { conferenceCount, talksCount } }) => (
+const getLinks = (talksCount, conferenceCount, role) => {
+  const links = [];
+
+  links.push({ to: upcoming, title: 'Upcoming' });
+  links.push({ to: past, title: 'Past' });
+
+  if (talksCount) links.push({ to: myTalks, title: 'My talks' });
+  if (conferenceCount) links.push({ to: myEvents, title: 'MyEvents' });
+  if (role === admin) {
+    links.push({ to: talks, title: 'Talks' });
+    links.push({ to: manageUsers, title: 'Manage users' });
+  }
+
+  return links.map(({ to, title }) => (
+    <NavLink
+      exact
+      to={to}
+      key={to}
+      className="tabs-list__anchor"
+      activeClassName="tabs-list__anchor_active"
+    >
+      {title}
+    </NavLink>
+  ));
+};
+
+const Tabs = ({
+  talksCount, conferenceCount, role,
+}) => (
   <div className="tabs-layout">
     <div className="tabs-wrapper">
-      <AutorizedUserMenu
-        conferenceCount={conferenceCount}
-        talksCount={talksCount}
-      />
-      <Route
-        path={upcoming}
-        exact
-        component={Upcoming}
-      />
-      <Route
-        path={myTalks}
-        exact
-        component={TalksWrapper}
-      />
-      <Route
-        path={myEvents}
-        exact
-        component={MyEvents}
-      />
-      <Route
-        path={`${conference}/:id`}
-        exact
-        component={Conference}
-      />
+      <NavigationBar>
+        {
+          getLinks(talksCount, conferenceCount, role)
+        }
+      </NavigationBar>
+      <Switch>
+        <Route
```

Comment:
let's extract this list to separate variable. it is getting bigger and bigger seems to me